app/forms/flood_risk_engine/steps/base_redirectable_form.rb

```diff
@@ -0,0 +1,18 @@
+module FloodRiskEngine
+  module Steps
+    class BaseRedirectableForm < BaseForm
```

Comment:
The PR is not ready to be reviewed yet. I moved the label as I work on all the requested changes.

But No I won't remove it, as this was useful,and may prove useful